forc-plugins/forc-client/src/util/tx.rs

```diff
@@ -71,20 +81,29 @@ pub(crate) fn prompt_forc_wallet_password() -> Result<String> {
 
 pub(crate) fn check_and_create_wallet_at_default_path(wallet_path: &Path) -> Result<()> {
     if !wallet_path.exists() {
-        let question = format!("Could not find a wallet at {wallet_path:?}, would you like to create a new one? [y/N]: ");
-        let accepted = ask_user_yes_no_question(&question)?;
-        let new_options = New {
-            force: false,
-            cache_accounts: None,
-        };
-        if accepted {
-            new_wallet_cli(wallet_path, new_options)?;
-            println!("Wallet created successfully.");
-            // Derive first account for the fresh wallet we created.
-            new_at_index_cli(wallet_path, 0)?;
-            println!("Account derived successfully.");
-        } else {
-            anyhow::bail!("Refused to create a new wallet. If you don't want to use forc-wallet, you can sign this transaction manually with --manual-signing flag.")
+        let question =
+            format!("Could not find a wallet at {wallet_path:?}, please select an option: ");
+        let wallet_options = ask_user_with_options(
+            &question,
+            &["Create new wallet", "Import existing wallet"],
+            0,
+        )?;
+        match wallet_options {
+            0 => {
+                new_wallet_cli(wallet_path, New { force: false, cache_accounts: None })?;
+                println!("Wallet created successfully.");
+                // Derive first account for the fresh wallet we created.
```

Comment:
Not sure if it's worth it to change or keep, up to you for taste, but these 3 lines here and in the next branch arm can be done after the `match` since the `bail` won't do them.

Reply:
addressed: https://github.com/FuelLabs/sway/pull/6680/commits/9c042b7be80452fadf2aa45d27dccacc18923ac5